apm-agent-plugins/apm-aws-sdk/apm-aws-sdk-1-plugin/src/test/java/co/elastic/apm/agent/awssdk/v1/S3ClientIT.java

```diff
@@ -53,25 +52,79 @@ public void setupClient() {
     public void testS3Client() {
         Transaction transaction = startTestRootTransaction("s3-test");
 
-        executeTest("CreateBucket", BUCKET_NAME, () -> s3.createBucket(BUCKET_NAME));
-        executeTest("CreateBucket", NEW_BUCKET_NAME, () -> s3.createBucket(NEW_BUCKET_NAME));
-        executeTest("ListBuckets", null, () -> s3.listBuckets());
-        executeTest("PutObject", BUCKET_NAME, () -> s3.putObject(BUCKET_NAME, OBJECT_KEY, "This is some Object content"));
-        executeTest("ListObjects", BUCKET_NAME, () -> s3.listObjects(BUCKET_NAME));
-        executeTest("GetObject", BUCKET_NAME, () -> s3.getObject(BUCKET_NAME, OBJECT_KEY));
-        executeTest("CopyObject", NEW_BUCKET_NAME, () -> s3.copyObject(BUCKET_NAME, OBJECT_KEY, NEW_BUCKET_NAME, "new-key"));
-        executeTest("DeleteObject", BUCKET_NAME, () -> {
+        newTest(() -> s3.createBucket(BUCKET_NAME))
+            .operationName("CreateBucket")
+            .entityName(BUCKET_NAME)
+            .otelAttribute("aws.s3.bucket", BUCKET_NAME)
```

Comment:
[for reviewer] this tests the expected OTel attribute capture that was added with this PR.